src/main/java/org/sagebionetworks/bridge/services/Exporter3Service.java

```diff
@@ -61,10 +69,82 @@
     static final String CONFIG_KEY_SYNAPSE_TRACKING_VIEW = "synapse.tracking.view";
     static final String CONFIG_KEY_TEAM_BRIDGE_ADMIN = "team.bridge.admin";
     static final String CONFIG_KEY_TEAM_BRIDGE_STAFF = "team.bridge.staff";
-    static final String CONFIG_KEY_WORKER_SQS_URL = "workerPlatform.request.sqs.queue.url";
     static final String FOLDER_NAME_BRIDGE_RAW_DATA = "Bridge Raw Data";
+    static final String TABLE_NAME_PARTICIPANT_VERSIONS = "Participant Versions";
     static final String WORKER_NAME_EXPORTER_3 = "Exporter3Worker";
 
+    static final List<ColumnModel> PARTICIPANT_VERSION_COLUMN_MODELS;
+    static {
+        ImmutableList.Builder<ColumnModel> listBuilder = ImmutableList.builder();
+
+        // Original healthcodes are GUIDs. These are 36 characters long.
+        ColumnModel healthCodeColumn = new ColumnModel();
+        healthCodeColumn.setName("healthCode");
+        healthCodeColumn.setColumnType(ColumnType.STRING);
+        healthCodeColumn.setMaximumSize(36L);
+        listBuilder.add(healthCodeColumn);
+
+        ColumnModel participantVersionColumn = new ColumnModel();
+        participantVersionColumn.setName("participantVersion");
+        participantVersionColumn.setColumnType(ColumnType.INTEGER);
+        listBuilder.add(participantVersionColumn);
+
+        ColumnModel createdOnColumn = new ColumnModel();
+        createdOnColumn.setName("createdOn");
+        createdOnColumn.setColumnType(ColumnType.DATE);
+        listBuilder.add(createdOnColumn);
+
+        ColumnModel modifiedOnColumn = new ColumnModel();
+        modifiedOnColumn.setName("modifiedOn");
+        modifiedOnColumn.setColumnType(ColumnType.DATE);
+        listBuilder.add(modifiedOnColumn);
+
+        // We have pre-existing validation that checks that the total length of all data groups is less than
+        // 250 characters. This validation predates the use of string lists in Synapse. Additionally, some apps have
+        // 30+ data groups, and some apps have data groups with 20+ characters. Synapse table row width is valuable, so
+        // instead of reserving potentially 600+ characters that will be sparsely used, we will keep to the
+        // 250 character limit.
+        ColumnModel dataGroupsColumn = new ColumnModel();
+        dataGroupsColumn.setName("dataGroups");
+        dataGroupsColumn.setColumnType(ColumnType.STRING);
+        dataGroupsColumn.setMaximumSize(250L);
+        listBuilder.add(dataGroupsColumn);
+
+        // These are 2-character language codes (eg "en", "es"). Let's generously allow participants to export up to
+        // 10 languages, which is 20 characters at most, and should be more than enough for nearly all participants.
```

Comment:
I don't think this changes your calculations, but at some point we'll probably go to the five character version that specifies region, e.g. en-US and en-UK.

Reply:
Bumped this up to 50 characters.